Packs/Core/Playbooks/playbook-External_Login_Password_Spray_README.md

```diff
@@ -9,16 +10,21 @@ This playbook is designed to handle the following alerts:
 The playbook is designed to investigate and respond to external login password sprays. It enriches the external IP to enable early containment, retrieves event information, and determines how the attack was carried out and whether it was successful.
 
 Playbook Stages:
```

Comment:
You forgot indentations
It should look like that:

The playbook executes the following stages:

Early Containment:
- With analyst approval, the playbook will block the malicious external IP address involved in the password spray attack, limiting the attacker's ability to continue their actions.

Investigation:
- The playbook analyzes the timestamps of the login attempts to detect patterns, checks whether any logons were successful, and retrieves the Risk Score for users who successfully logged in as part of the attack.

Containment:
- Based on the user’s risk level, the playbook will expire the user’s password to prevent further unauthorized access and terminate any active RDP sessions for the affected user.

Reply:
please see now